config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -4,7 +4,7 @@ periodics:
   annotations:
     fork-per-release: "true"
     fork-per-release-periodic-interval: 1h 2h 6h 24h
-    fork-per-release-generic-suffix: "true"
+    testgrid-create-test-group: "true"
```

Comment:
TestGrid (specifically [Configurator](https://github.com/kubernetes/test-infra/blob/c86562b1a78ba52ab98749204a52e960f4a79b33/testgrid/cmd/configurator/prow.go#L55)) will automatically create a test group for you if the job either:
- is a postsubmit or periodic, or
- has a `testgrid-dashboards` entry already

These lines don't hurt anything, but they're redundant.